src/classes/EP_ManageEPTemplate_CTRL.cls

```diff
@@ -455,10 +455,11 @@ public with sharing class EP_ManageEPTemplate_CTRL {
         Component.Apex.OutputPanel formPanel = new Component.Apex.OutputPanel();
     
         //creates a box around the form with a label for the hierarchy level, and a color bar to more easily grock which level we're at
+        // uses escape = false but the labels are escaped
```

Comment:
Is there more this comment should state as to why the labels are escaped, but `escape=false` is included on the component?

Reply:
@force2b - yeah I just wanted to add some comments so that future readers who see the escape = false know that it was intentional but that we escaped the inputs.  Thanks